modules/apps/document-library/document-library-repository-cmis-impl/src/main/java/com/liferay/document/library/repository/cmis/internal/CMISRepository.java

```diff
@@ -1519,6 +1520,11 @@ public boolean verifyInheritableLock(long folderId, String lockUuid) {
 		throw new UnsupportedOperationException();
 	}
 
+	@Deactivate
```

Comment:
Hey @regiCesar21, this class is not an osgi service. It won't work. The solution here is just merge the `CMISSessionCache` into its only usage.